test/integration/default/verify/controls/aws_flow_log.rb

```diff
@@ -0,0 +1,95 @@
+iam_fixtures = {}
+[
+  'flow_log_default_subnet_id',
+  'flow_log_default_vpc_id',
+  'log_metric_filter_1_log_group_name'
```

Comment:
If you need a log group for you flow logs, you should create one just for the flow logs.  Don't re-use the ones used for LMF testing.  That creates unexpected tight couplings between test suites, and means we can run them independently.